homeassistant/components/isy994/__init__.py

```diff
@@ -152,7 +152,7 @@ async def async_setup_entry(
         https = False
         port = host.port or 80
         session = aiohttp_client.async_create_clientsession(
-            hass, verify_ssl=False, cookie_jar=CookieJar(unsafe=True)
+            hass, ssl_context=False, cookie_jar=CookieJar(unsafe=True)
```

Comment:
The behavior here is a bit unexpected. If ssl_context is false it seems to imply a non secure connection